Lib/test/test_pdb.py

```diff
@@ -4441,6 +4441,30 @@ def test_multiline_completion(self):
 
         self.assertIn(b'42', output)
 
+    def test_multiline_indent_completion(self):
+        script = textwrap.dedent("""
+            import pdb; pdb.Pdb().set_trace()
+        """)
+
+        # \t should always complete a 4-space indent
+        # This piece of code will raise an IndentationError or a SyntaxError
+        # if the completion is not working as expected
+        input = b"def func():\n"
+        input += b"\ta = 1\n"
+        input += b" \ta += 1\n"
+        input += b"  \ta += 1\n"
+        input += b"   \tif a > 0:\n"
+        input += b"        a += 1\n"
+        input += b"\t\treturn a\n"
+        input += b"\n"
+        input += b"func()\n"
+        input += b"c\n"
```

Comment:
Could this be a multi-line string?

```suggestion
        input = b"""\
def func():
\ta = 1
 \ta += 1
  \ta += 1
   \tif a > 0:
        a += 1
\t\treturn a

func()
c
"""
```

Reply:
I don't really like the left-aligned string so I did something similar to keep a single string-like constant.